pages/profile/[vin].js

```diff
@@ -0,0 +1,65 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+import Cars from "../../db/db.json";
+export default function CarDetails() {
+  const router = useRouter();
+  const { vin } = router.query;
+  const activeCar = Cars.find((car) => {
+    console.log(car.VIN);
+    console.log(vin);
```

Comment:
Please remove console.logs that help you during development from production code